src/runtime/rpc/rpc_session.h

```diff
@@ -48,6 +48,7 @@ enum class RPCCode : int {
   kModuleFree,
   kModuleGetFunc,
   kModuleGetSource,
+  kTestRemoteOpenGL,
```

Comment:
this can be safely removed when we upstream?

Reply:
Yes, this is just for my debugging purposes. Will remove when I cleanup.

Reply:
Deleted.